src/utils/pattern.spec.ts

```diff
@@ -82,6 +82,8 @@ describe('Utils → Pattern', () => {
 				assert.ok(util.isDynamicPattern('{a,}'));
 				assert.ok(util.isDynamicPattern('{,b}'));
 				assert.ok(util.isDynamicPattern('{a,b}'));
+				assert.ok(util.isDynamicPattern('{a,b,c}'));
+				assert.ok(util.isDynamicPattern('{a' + ','.repeat(999999) + 'b}'));
```

Comment:
I believe we can add tests for nested brackets right now to rule out future breakdowns.

```suggestion
				assert.ok(util.isDynamicPattern('{a' + ','.repeat(999999) + 'b}'));
				assert.ok(util.isDynamicPattern('{a,b,{c,d}}'));
				// The second braces pass
				assert.ok(util.isDynamicPattern('{a,b,{c,d}'));
```

Reply:
Done!